### ISSUE TITLE
SRFI 27 improvement

### ISSUE THREAD

**Issue by jasonmacduffie** [justinethier/cyclone#230]
I think it's worthwhile to replace my LCG SRFI 27 implementation with the reference implementation provided by mrg32k3a-b.c. I will begin working on this.

**Comment by justinethier** [comment]
That's a great idea, thanks.

There is some code in the runtime that may be useful here: https://github.com/justinethier/cyclone/blob/master/runtime.c#L5592

**Comment by arthurmaciel** [comment]
Hi! Is the current implementation good enough or does it still need improvements?

**Comment by jasonmacduffie** [comment]
Yes, currently it's using an LCG I believe. It would be better to use the
reference code provided in the SRFI.

**Comment by justinethier** [comment]
@jasonmacduffie Is that a change you are still interested in developing?

**Comment by jasonmacduffie** [comment]
Not especially, I haven't touched Cyclone in a while. I think it should be
a straightforward change. Just haven't found the time for coding lately but
maybe I'll give it a shot if nobody else is.
